django/forms/forms.py

```diff
@@ -152,7 +168,9 @@ def __getitem__(self, name):
             raise KeyError(
                 "Key %r not found in '%s'" % (name, self.__class__.__name__))
         if name not in self._bound_fields_cache:
-            self._bound_fields_cache[name] = BoundField(self, field, name)
+            self._bound_fields_cache[name] = BoundField(
+                self, field, name, self.renderer,
```

Comment:
Do we actually need to pass in the renderer separately to `BoundField`? It gets a reference to the whole form already, so it could just access `form.renderer`.